doc/classes/SkeletonIK.xml

```diff
@@ -26,12 +28,14 @@
 			<argument index="0" name="one_time" type="bool" default="false">
 			</argument>
 			<description>
+                Starts applying IK effects on each frame to the [Skeleton] bones but will only take effect starting on the next frame. If used with a bool [code]one_time[/code] parameter will take effect immediately but also reset on next frame.
```

Comment:
```suggestion
                Starts applying IK effects on each frame to the [Skeleton] bones but will only take effect starting on the next frame. If [code]one_time[/code] is [code]true[/code], this will take effect immediately but also reset on the next frame.
```